Ecommerce/app/controllers/categories_controller.rb

```diff
@@ -2,6 +2,9 @@ class CategoriesController < ApplicationController
   before_action :authenticate_user!
 
   def index
+  end
+
+  def show
```

Comment:
In the show request of category we are showing all items?

Reply:
No, We are showing according to the category selected (Male or Female)